Don't create a PR if all changesets are empty (#206)
changesets/action: .changeset/three-needles-protect.md

---
"@changesets/action": patch
---

> Don't create a PR if all changesets are empty

Reviewer: ```suggestion
Skip creating a PR when all existing changesets are empty.
```
